apps/testingaccessibility/src/components/portable-text/index.tsx

```diff
@@ -214,11 +226,11 @@ const PortableTextComponents: PortableTextComponentsProps = {
   },
   types: {
     bodyVideo: ({value}: BodyVideoProps) => {
-      const {url, title, caption} = value
+      const {url, title, caption, videoResource} = value
       return (
         <figure className="video">
           <VideoProvider>
-            <Video url={url} title={title} />
+            <Video url={url} title={title} videoResource={videoResource} />
```

Comment:
I'd prefer to pull the title and media url from the video resource, but that's a broader change.